### PR TITLE
Add Print Service

### PR DESCRIPTION
**Pull request by bharatkashyap**
Adding [Print Service ](https://github.com/Sunbird-Ed/program-service/pull/124/files#diff-a07fb9935cf091d4c8a336bf822ca4cb9d85701751a0ba5672269e0ce0314839 ) to Kong

### CODE REVIEW

**Comment by bharatkashyap** [comment]
@keshavprasadms Requesting review

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5945,6 +5945,24 @@ kong_apis:
...
+    request_path: "{{ program_service_prefix }}/v1/print/pdf"
+    upstream_url: "{{ program_service_url }}/program/v1/print/pdf"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'programAccess'
```
Comment (keshavprasadms) [inline ansible/roles/kong-api/defaults/main.yml:5959]: @pallakartheekreddy @bharatkashyap Does the front portal service need access to this API? Will this API get invoked from portal service in any scenarios?

Comment (bharatkashyap) [reply ansible/roles/kong-api/defaults/main.yml:5959]: Yes @keshavprasadms this API is to be invoked from the frontend portal

Comment (keshavprasadms) [reply ansible/roles/kong-api/defaults/main.yml:5959]: In that case, please add thie ACL under this consumer - https://github.com/project-sunbird/sunbird-devops/blob/dock-0.0.3/ansible/roles/kong-consumer/defaults/main.yml#L30

Comment (bharatkashyap) [reply ansible/roles/kong-api/defaults/main.yml:5959]: Have done @keshavprasadms

Comment (bharatkashyap) [reply ansible/roles/kong-api/defaults/main.yml:5959]: Have done @keshavprasadms

**Comment by keshavprasadms** [comment]
Thanks @bharatkashyap 
@gandham-santhosh @SMYALTAMASH Please review this PR and provide approval to onboard this API to vdn dev.

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (gandham-santhosh) [reply ansible/roles/kong-api/defaults/main.yml:5959]: @bharatkashyap @pallakartheekreddy as per process any new API need to be reviewed by DC. Can you please point to any design document/confluence page mentioning the usage of this API and being reviewed by DC

Comment (bharatkashyap) [reply ansible/roles/kong-api/defaults/main.yml:5959]: Hi @gandham-santhosh [This ](https://project-sunbird.atlassian.net/wiki/spaces/PRD/pages/1978138627/Question+Paper+Creation+Sourcing+and+Printing)is the Product Requirements Document for printing Question Papers which mentions this API. 


cc: @SaketSinha

Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml:5959]: Saket can you update the API Sheet with the API Details?

**Review by SMYALTAMASH — APPROVED** [review]

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (SaketSinha) [reply ansible/roles/kong-api/defaults/main.yml:5959]: @gandham-santhosh - Please refer to section [6.3 - Print Preview - For Question Paper creator (Sourcing Reviewer)](https://project-sunbird.atlassian.net/wiki/spaces/PRD/pages/1978138627/Question+Paper+Creation+Sourcing+and+Printing) in the confluence document.

More decision on creating this API, please refer to the discussion [here](https://github.com/project-sunbird/sunbird-community/discussions/9#discussioncomment-304715).

Do let me know if any thing else is required as well.

Comment (SaketSinha) [reply ansible/roles/kong-api/defaults/main.yml:5959]: Hi Team, there is a gap in understanding, we are not aware of the steps needed to be done, please share a brief.
- DC Review?
- API sheet?

Comment (vrayulu) [reply ansible/roles/kong-api/defaults/main.yml:5959]: This API need not be available on public api-manager. Please onboard only on the internal API manager. Creation-portal backend should invoke this service using internal route. 

@pallakartheekreddy - portal backend should expose the path for UI (and protected by keycloak auth) to call this internal API. Please help @bharatkashyap to get it implemented that way.

Comment (pallakartheekreddy) [reply ansible/roles/kong-api/defaults/main.yml:5959]: @bharatkashyap As discussed, Please use the below URL while making an API call from UI
`{{host}}/content/program/v1/print/pdf`
cc: @vrayulu

**Pull request closed without merging**